docroot/modules/custom/dcc_gtd_registration/dcc_gtd_registration.info.yml

```diff
@@ -0,0 +1,5 @@
+name: Register
+type: module
+description: This is a multi step form for registering
```

Comment:
Please change the description to "Defines a multistep form for registering to global training days."